README.md

```diff
@@ -18,63 +17,41 @@ In this version of _Adagrams_, we will only be working with the English alphabet
 
 ![an image of a pile of letter tiles](images/letter-tiles.jpeg)
 
-## Pair Programming
```

Comment:
I can't throw a comment on it since it isn't changed, but in the first paragraph of the "Goal" section above, there's a sentence that starts with "In this project you and your partner..." that we should update.